### PR TITLE
Create sanidhya.json

### PR DESCRIPTION
**Pull request by codersonet**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by CoderSoNet** [comment]
May i need to change record bro??

File: domains/sanidhya.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  "Description": "This is my personal Website of portfolio",
+
+  "repo": "https://github.com/CoderSoNet/codersonetwebsite.github.io",
+
+  "owner": {
+
+    "username": "CoderSoNet",
+
+    "email": "contactus@sanidhya.is-a.dev",
```
Comment (WilliamDavidHarrison) [inline domains/sanidhya.json:11]: Sorry, this email doesn't exist as the domain doesn't have email.

File: domains/sanidhya.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    "twitter": "https://twitter.com/ShanuUniyal_On",
+
+    "discord": "<@781127625263415326>"
```
Comment (WilliamDavidHarrison) [inline domains/sanidhya.json:15]:
```suggestion
    "discord": "781127625263415326"
```

File: domains/sanidhya.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  "repo": "https://github.com/CoderSoNet/codersonetwebsite.github.io",
+
+  "owner": {
+
+    "username": "CoderSoNet",
+
+    "email": "contactus@sanidhya.is-a.dev",
+
+    "twitter": "https://twitter.com/ShanuUniyal_On",
```
Comment (WilliamDavidHarrison) [inline domains/sanidhya.json:13]:
```suggestion
    "twitter": "ShanuUniyal_On",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**